# SOME DESCRIPTIVE TITLE.
# Copyright (C) YEAR ORGANIZATION
# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
#
msgid ""
msgstr ""
"Project-Id-Version: \n"
"POT-Creation-Date: 2017-06-02 00:20+0300\n"
"PO-Revision-Date: 2017-06-02 00:20+0300\n"
"Language-Team: \n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Generated-By: pygettext.py 1.5\n"
"X-Generator: Poedit 1.8.7.1\n"
"Last-Translator: \n"
"Plural-Forms: nplurals=2; plural=(n != 1);\n"
"Language: en\n"

#: VM/BDL/BDCon.py:19 VM/UI/ChoosingItem.py:41
msgid "There are no items to sell"
msgstr "There are no items to sell"

#: VM/BDL/BDCon.py:20 VM/UI/ChoosingItem.py:42
msgid "Message"
msgstr "Message"

#: VM/BDL/DataSources/MySQLDB.py:60 VM/BDL/DataSources/MySQLDB.py:76
#: VM/BDL/DataSources/MySQLDB.py:91 VM/BDL/DataSources/MySQLDB.py:107
#: VM/BDL/DataSources/MySQLDB.py:123 VM/BDL/DataSources/MySQLDB.py:178
#: VM/UI/CheckPass.py:31 VM/Vending.py:40
msgid "Error"
msgstr "Error"

#: VM/BDL/DataSources/MySQLDB.py:60 VM/BDL/DataSources/MySQLDB.py:91
#: VM/BDL/DataSources/MySQLDB.py:107 VM/BDL/DataSources/MySQLDB.py:123
msgid "Database connection failed"
msgstr "Database connection failed"

#: VM/BDL/DataSources/MySQLDB.py:76 VM/UI/CheckPass.py:31
msgid "There is errors in the configuration file. No section."
msgstr "There is errors in the configuration file. No section."

#: VM/BDL/DataSources/MySQLDB.py:178
msgid "Database error"
msgstr "Database error"

#: VM/Errors.py:19
msgid "Close"
msgstr "Close"

#: VM/KP/KPProvider.py:91
msgid "Error in recognition of note"
msgstr "Error in recognition of note"

#: VM/KP/KPProvider.py:106
msgid "Protocol version's setting error"
msgstr "Protocol version's setting error"

#: VM/KP/KPProvider.py:113
msgid "Error in request of notes' channels"
msgstr "Error in request of notes' channels"

#: VM/KP/KPProvider.py:122
msgid "Error in notes' denomination reading"
msgstr "Error in notes' denomination reading"

#: VM/KP/KPProvider.py:129
msgid "Error in setting channels of notes receiving"
msgstr "Error in setting channels of notes receiving"

#: VM/KP/KPProvider.py:136
msgid "Engage procedure error"
msgstr "Engage procedure error"

#: VM/KP/KPProvider.py:151
msgid "Device synchronization failed"
msgstr "Device synchronization failed"

#: VM/KP/KPProvider.py:288
msgid "Note receiver's port is not found"
msgstr "Note receiver's port is not found"

#: VM/KP/KPProvider.py:305
msgid "Devise not responding"
msgstr "Devise not responding"

#: VM/UI/ChoosingItem.py:95
msgid "Select the model"
msgstr "Select the model"

#: VM/UI/ChoosingItem.py:96 VM/UI/ReceiveCash.py:50
msgid "Cancel"
msgstr "Cancel"

#: VM/UI/FinishWindow.py:33
msgid "The key was written successfully"
msgstr "The key was written successfully"

#: VM/UI/FinishWindow.py:34
msgid "Thanks for purchase"
msgstr "Thanks for purchase"

#: VM/UI/GivingOutItem.py:29
msgid "Wait for delivery..."
msgstr "Wait for delivery..."

#: VM/UI/GivingOutItem.py:36
msgid "Delivery error. Call techsupport, please."
msgstr "Delivery error. Call techsupport, please."

#: VM/UI/ReceiveCash.py:47
msgid "Incomming cash"
msgstr "Incomming cash"

#: VM/UI/ReceiveCash.py:48
msgid "Pay"
msgstr "Pay"

#: VM/UI/ReceiveCash.py:49
msgid "Mashine does not give any odd money"
msgstr "Mashine does not give any odd money"

#: VM/UI/ReceiveCash.py:51
msgid "Next"
msgstr "Next"

#: VM/UI/ScanBrelok.py:70
msgid "Press the button \"Scan\""
msgstr "Press the button \"Scan\""

#: VM/UI/ScanBrelok.py:71
msgid "and enclose your key"
msgstr "and enclose your key"

#: VM/UI/ScanBrelok.py:72
msgid "to the scanner"
msgstr "to the scanner"

#: VM/UI/ScanBrelok.py:73
msgid "Scanning..."
msgstr "Scanning..."

#: VM/UI/ScanBrelok.py:74
msgid "Error!"
msgstr "Error!"

#: VM/UI/ScanBrelok.py:75
msgid "Scan"
msgstr "Scan"

#: VM/UI/WriteBrelok.py:40
msgid "The key is given away"
msgstr "The key is given away"

#: VM/UI/WriteBrelok.py:41
msgid "Press 'Next'"
msgstr "Press 'Next'"

#: VM/UI/WriteBrelok.py:42
msgid "and enclose it to the scanner"
msgstr "and enclose it to the scanner"

#: VM/UI/WriteBrelok.py:43
msgid "Writing..."
msgstr "Writing..."

#: VM/UI/WriteBrelok.py:44
msgid "Writing failed!"
msgstr "Writing failed!"

#: VM/UI/WriteBrelok.py:45
msgid "Try again, please."
msgstr "Try again, please."

#: VM/UI/WriteBrelok.py:46
msgid "Write"
msgstr "Write"

#: VM/Vending.py:39
msgid "Database connection error"
msgstr "Database connection error"
